go-controller/pkg/node/node.go

```diff
@@ -155,6 +155,21 @@ func isOVNControllerReady(name string) (bool, error) {
 	return true, nil
 }
 
+// Starting with v21.03.0 OVN sets OVS.Interface.external-id:ovn-installed
+// and OVNSB.Port_Binding.up when all OVS flows associated to a
+// logical port have been successfully programmed.
+func getOVNIfUpCheckMode() cni.PodCheckMode {
+	checkMode := cni.PodCheckMode{
+		UseOVSExternalIDs: true,
+	}
+
+	if _, _, err := util.RunOVNSbctl("--columns=up", "list", "Port_Binding"); err != nil {
```

Comment:
do we need to distinguish here between an error here reading the value, vs it not existing? Otherwise if there is some transient error reading, we would just assume there is no support? Maybe look for some substring of the error being:
```
ovn-sbctl: Port_Binding does not contain a column whose name matches "up"
```

Reply:
Ack, makes sense.